### DISCUSSION TITLE
essay/2022/01/05/daily-coding

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#125] [Comments]
# essay/2022/01/05/daily-coding

jQuery를 만든 존 레식이 언젠가 매일매일 코딩하기라는 글을 쓴 적이 있습니다.

https://jeho.page/essay/2022/01/05/daily-coding.html

**Comment by IAMISTP** [comment]
안녕하세요 프론트엔드 개발자로 취업하고자 현재 취준중인 취준생입니다.
혼자 공부를 하다보니 제가 하고 있는게 맞는건지 제 선택에 확신이 생기지 않아 나름 계획을 세워 하는데도 
이게 맞는건가 하는 항상 의구심만 계속 들었어요
그러다보니 현재 상황에 회피 하게 되고 , 변명이 늘며 늘어지는것 같더라구요
요즘 특히 더 주위에서 프론트 엔드 개발로 취업하기 위해 이력서 130곳을 지원하였는데 한곳만 연락왔다는 지인의 이야기를 들으니 더욱더 생각이 많아지더라구요,, 그런데 유튜브를 통해서 저자님의 말씀을 듣고 많았던 생각들이 조금은 정리가 되었습니다.
꾸준히만 저한테도 취업을 할수 있는 기회가 오지 않을까 하구요.
제 꿈도 저자님처럼 제 서비스를 만들어보는게 꿈입니다!
힘들때마다 작성해주신 블로그 글들을 매일 읽으러 오겠습니다.
감사합니다!

**Reply by BenjaminKim** [reply]
제가 힘이 될 수 있다니 기쁩니다.
꾸준히 열심히 하는 사람 중에 잘 안된 사람을 본 적이 없는 것 같습니다.
매일 정진하시다가 좋은 곳에 취업하시고... 언젠가 그래 그럴 때가 있었지 하는 날이 오길 바랍니다.

**Reply by IAMISTP** [reply]
감사합니다ㅠㅠㅠ

**Comment by dm-jeong** [comment]
안녕하세요.
좋은 글 감사합니다.

매일 꾸준히 프로그래밍할 때 어떤 기준으로 프로그래밍을 하시나요?
라인수를 기준으로 하시나요?
시간 간격으로 하시나요?

조언 부탁드립니다!

**Reply by BenjaminKim** [reply]
task 단위로 일을 하는데 하루에 최소한 1개 task 는 하려고 노력하고 있습니다.
task 가 뭔가에 따라 다른데, 5분만에 할 수 있는 것도 있고 1시간 넘게 걸리는 일들도 있고 다양하네요.

관련글: [할일/일정 관리 어떻게 하세요?](https://jeho.page/essay/2023/12/19/todo.html)

**Reply by dm-jeong** [reply]
답변 감사합니다!
